.pre-commit-config.yaml

```diff
@@ -1,24 +1,66 @@
+exclude: |
+  (?x)
+  # NOT INSTALLABLE ADDONS
+  # END NOT INSTALLABLE ADDONS
+  # Files and folders generated by bots, to avoid loops
+  ^setup/|/static/description/index\.html$|
+  # We don't want to mess with tool-generated files
+  .svg$|/tests/([^/]+/)?cassettes/|^.copier-answers.yml$|^.github/|
+  # Maybe reactivate this when all README files include prettier ignore tags?
+  ^README\.md$|
+  # Library files can have extraneous formatting (even minimized)
+  /static/(src/)?lib/|
+  # Repos using Sphinx to generate docs don't need prettying
+  ^docs/_templates/.*\.html$|
+  # You don't usually want a bot to modify your legal texts
+  (LICENSE.*|COPYING.*)|
+  # Exclude external addons
+  odoo/src/ext_addons/|
+  # Exclude container config file, especially the generated ones
+  ^odoo/*.*
+default_language_version:
+  python: python3
+  node: "14.18.0"
 repos:
+  - repo: local
+    hooks:
+      # These files are most likely copier diff rejection junks; if found,
+      # review them manually, fix the problem (if needed) and remove them
+      - id: forbidden-files
+        name: forbidden files
+        entry: found forbidden files; remove them
+        language: fail
+        files: "\\.rej$"
+  - repo: https://github.com/myint/autoflake
+    rev: v1.4
+    hooks:
+      - id: autoflake
+        args:
+          - --expand-star-imports
+          - --ignore-init-module-imports
+          - --in-place
+          - --remove-all-unused-imports
+          - --remove-duplicate-keys
+          - --remove-unused-variables
   - repo: https://github.com/psf/black
-    rev: 20.8b1
+    rev: 21.9b0
```

Comment:
This black version has currently heavy problems with the current click version. I guess some of the CI for the OCA might be also influenced by this.

```
Traceback (most recent call last):
  File "/home/florian/.cache/pre-commit/repod82ojeyu/py_env-python3/bin/black", line 8, in <module>
    sys.exit(patched_main())
  File "/home/florian/.cache/pre-commit/repod82ojeyu/py_env-python3/lib/python3.8/site-packages/black/__init__.py", line 1282, in patched_main
    patch_click()
  File "/home/florian/.cache/pre-commit/repod82ojeyu/py_env-python3/lib/python3.8/site-packages/black/__init__.py", line 1268, in patch_click
    from click import _unicodefun  # type: ignore
ImportError: cannot import name '_unicodefun' from 'click' (/home/florian/.cache/pre-commit/repod82ojeyu/py_env-python3/lib/python3.8/site-packages/click/__init__.py)
```

https://github.com/psf/black/issues/2964